src/agent/solana/exporter.rs

```diff
@@ -468,13 +474,24 @@ impl Exporter {
                "publish_pubkey" => publish_keypair.pubkey().to_string(),
         );
 
+        let now = Local::now();
+
         // Filter out price accounts we're not permissioned to update
         Ok(fresh_updates
             .into_iter()
             .filter(|(id, _data)| {
                 let key_from_id = Pubkey::from((*id).clone().to_bytes());
-                if self.our_prices.contains(&key_from_id) {
-                    true
+                if let Some(market_hours) = self.our_prices.get(&key_from_id) {
+                    let ret = market_hours.can_publish_at(&now);
+
+		    if !ret {
+			debug!(self.logger, "Exporter: Attempted to publish price outside market hours";
```

Comment:
The formatting is bad here again because of mixed tab and spaces

Reply:
Guilty as charged, I had a fairly old nightly locally while our CI grabs a fresh one with each run.

Reply:
there's a way to pin the version used for formatting which solves this problem. I forget exactly how to do it but we're doing it in pyth-crosschain.